services/app/src/pages/game/Game.jsx

```diff
@@ -2,11 +2,26 @@ import React from 'react';
 
 const Room = () => {
   const currentGameURL = window.location.href;
+  const gameURL =
```

Comment:
What is the difference between `currentGameURL` and `gameIRL`? Can it be reused?